<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCollectionDialog</class>
 <widget class="QDialog" name="AddCollectionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>257</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add new collection</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="_3">
   <item>
    <layout class="QGridLayout" name="_4">
     <item row="1" column="1">
      <widget class="QLineEdit" name="editDatabase">
       <property name="toolTip">
        <string>Collection database file name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QToolButton" name="btnDatabase">
       <property name="toolTip">
        <string>Open file dialog</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../icons/light/resources/light.qrc">
         <normaloff>:/icons/database.svg</normaloff>:/icons/database.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblDatabase">
       <property name="text">
        <string>Database file name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName">
       <property name="toolTip">
        <string>Collection title</string>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblArchive">
       <property name="text">
        <string>Archive folder</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editArchive">
       <property name="toolTip">
        <string>Collection archive folder</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QToolButton" name="btnArchive">
       <property name="toolTip">
        <string>Open folder dialog</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../icons/light/resources/light.qrc">
         <normaloff>:/icons/open-folder.svg</normaloff>:/icons/open-folder.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QToolButton" name="btnSetDefaultName">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Retrieve collection name from index file</string>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="../../icons/light/resources/light.qrc">
         <normaloff>:/icons/set-default.svg</normaloff>:/icons/set-default.svg</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="options" native="true">
     <layout class="QVBoxLayout" name="_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBoxAddUnindexedBooks">
        <property name="text">
         <string>Add unindexed books</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_6">
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxMarkUnindexedAdDeleted">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Mark unindexed books as deleted</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="_7">
        <item>
         <widget class="QCheckBox" name="checkBoxAddMissingBooks">
          <property name="text">
           <string>Add books missing from archives</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblDefaultArchiveType">
          <property name="text">
           <string>Archive type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDefaultArchiveType">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxScanUnindexedArchives">
        <property name="text">
         <string>Scan unindexed archives</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_5" stretch="0,1,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblError">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="text">
        <string>Add</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>btnDatabase</tabstop>
  <tabstop>btnArchive</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnCancel</tabstop>
  <tabstop>editDatabase</tabstop>
  <tabstop>editArchive</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons/light/resources/light.qrc"/>
  <include location="../../icons/light/resources/light.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBoxAddUnindexedBooks</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxMarkUnindexedAdDeleted</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>324</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAddMissingBooks</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxDefaultArchiveType</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
